app/index.js

```diff
@@ -1,20 +1,26 @@
 import React from 'react';
 import {render} from 'react-dom';
 import {Provider} from 'react-redux';
+import { Route, Router, hashHistory, browserHistory} from 'react-router'
 
 import configureStore from './store/configureStore';
-
 import {productName, version} from '../package.json';
 
-const store = configureStore();
+import Login from './components/Login.react'
+import Configuration from './components/Configuration.react'
+import Status from './components/Oauth.react'
 
-import ConfigurationPage from './containers/ConfigurationPage';
+const store = configureStore();
 
 window.document.title = `${productName} v${version}`;
 
 render(
     <Provider store={store}>
-        <ConfigurationPage/>
+      <Router history={browserHistory}>
+        <Route exact path='/' component={Login} />
+        <Route path='/database-connector' component={Configuration} />
```

Comment:
while we're here, I think I'd prefer the URLs:
```
- `/login` for the login route
- `/` for the main tool (redirects to `/login` if necessary)
```

Reply:
Updated.. 

For redirect to `/login`-- I would add it as a part of authentication implementation.